### PR TITLE
I think I can update?

### PR DESCRIPTION
**Pull request by Jimmyson**
Referencing Issue #634,
I am not sure this implementation would update the file status for files added in where AniDB already has them listed.

Note: This also includes some SQL updates to missing indexes

### CODE REVIEW

**Comment by da3dsoul** [comment]
Make sure to update the required version at the top of each database file

File: Shoko.Server/Commands/AniDB/CommandRequest_SyncMyList.cs [hunk 1]
```diff
@@ -99,7 +99,18 @@ public override void ProcessCommand()
 
                     int fileID = dictAniFiles[vid.Hash].FileID;
                     // Is it in MyList
-                    if (onlineFiles.ContainsKey(fileID)) continue;
+                    if (onlineFiles.ContainsKey(fileID))
+                    {
+                        Raw_AniDB_MyListFile file = onlineFiles[fileID];
+                        
+                        // Update file state if deleted
+                        if (fileID.State = (int) AniDBFile_State.Deleted)
```
Comment (da3dsoul) [inline Shoko.Server/Commands/AniDB/CommandRequest_SyncMyList.cs]: == equality operator

Comment (Jimmyson) [inline Shoko.Server/Commands/AniDB/CommandRequest_SyncMyList.cs]: :facepalm:

Comment (da3dsoul) [inline Shoko.Server/Commands/AniDB/CommandRequest_SyncMyList.cs]: yep it happens to us all

**Comment by da3dsoul** [comment]
Now just the DB versions... Hopefully

**Comment by Jimmyson** [comment]
I'm still getting errors on the Enum from appveyor.

File: Shoko.Server/Commands/AniDB/CommandRequest_SyncMyList.cs [hunk 2]
```diff
@@ -99,7 +99,18 @@ public override void ProcessCommand()
 
                     int fileID = dictAniFiles[vid.Hash].FileID;
                     // Is it in MyList
-                    if (onlineFiles.ContainsKey(fileID)) continue;
+                    if (onlineFiles.ContainsKey(fileID))
+                    {
+                        Raw_AniDB_MyListFile file = onlineFiles[fileID];
+                        
+                        // Update file state if deleted
+                        if (fileID.State == (int) AniDBFile_State.Deleted)
```
Comment (da3dsoul) [inline Shoko.Server/Commands/AniDB/CommandRequest_SyncMyList.cs]: fileID...

Comment (Jimmyson) [inline Shoko.Server/Commands/AniDB/CommandRequest_SyncMyList.cs]: I spotted that too, the SQL commit will have that!

**Pull request merged**